sys-apps/smcipmitool/smcipmitool-2.19.0.171201.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver java-vm-2
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="SMCIPMITool"
+MY_PN_SRC_URI="SMCIPMItool"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="An out-of-band utility for interfacing with SuperBlade and IPMI devices via CLI"
+HOMEPAGE="https://www.supermicro.com/"
+SRC_URI="amd64? ( ftp://ftp.supermicro.com/utility/${MY_PN_SRC_URI}/Linux/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux_x64.tar.gz )
+	x86? ( ftp://ftp.supermicro.com/utility/${MY_PN_SRC_URI}/Linux/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux.tar.gz )"
+
+LICENSE="supermicro"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="sys-libs/ncurses:5
+	virtual/jre"
+
+RESTRICT="bindist fetch mirror strip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="opt/smcipmitool/libiKVM*.so
+	opt/smcipmitool/libjcurses*.so
+	opt/smcipmitool/libSharedLibrary*.so"
+
+pkg_nofetch() {
+	elog "Please download ${A} from"
+	elog "https://www.supermicro.com/SwDownload/UserInfo.aspx?sw=0&cat=IPMI"
+	elog "and place it in your DISTDIR directory."
+}
+
+src_install() {
+	# Choose ARCH
+	if use amd64; then
+		local my_arch="${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux_x64"
+	else
+		local my_arch="${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux"
+	fi
+
+	# Install binary
+	exeinto /opt/smcipmitool
+	doexe ${my_arch}/SMCIPMITool
+
+	# Install libs
+	exeinto /opt/smcipmitool
+	if use amd64; then
+		doexe ${my_arch}/*64.so
+	else
+		doexe ${my_arch}/libjcurses.* ${my_arch}/*32.so
```

Comment:
I'm a bit confused why jcurses is only needed on x86.

Reply:
Seems to a typo be me. There are libjcurses32.so and libjcurses64.so, so both are already installed by the other part. I will remove it.